UPGRADING

```diff
@@ -193,6 +193,11 @@ PHP 8.5 UPGRADE NOTES
     used by a cURL transfer. It is primarily useful when connection reuse or
     connection pooling logic is needed in PHP-level applications. When
     curl_getinfo() returns an array, this value is available as the "conn_id" key.
+  . Added support for CURLINFO_QUEUE_TIME_T (libcurl >= 8.6.0) to the curl_getinfo()
```

Comment:
```suggestion
  . Added support for CURLINFO_QUEUE_TIME_T to the curl_getinfo()
```

Very minor nit, we won't need to mention the libcurl version requirement twice. The one at the end should be do be sufficient.

Reply:
Hi @Ayesh 

Thank you for feedback!